pkg/edit/repl.go

```diff
@@ -0,0 +1,60 @@
+package edit
+
+// This file encapsulates functionality related to a complete REPL cycle. Such as capturing
+// information about the most recently executed interactive command.
+
+import (
+	"src.elv.sh/pkg/eval"
+	"src.elv.sh/pkg/eval/vals"
+	"src.elv.sh/pkg/eval/vars"
+	"src.elv.sh/pkg/parse"
+)
+
+//elvdoc:var after-command
+//
+// A list of functions to call after each interactive command completes. There is one pre-defined
+// function used to populate the [`$edit:command-duration`](./edit.html#editcommand-duration)
+// variable. Each function is called with a single [map](https://elv.sh/ref/language.html#map)
+// argument containing the following keys:
+//
+// * `code`: A string containing the code that was executed. If `is-file` is true this is the
+// content of the file that was sourced.
```

Comment:
Better just expose the `Source` struct, which is itself a pseudo-map and can be introspected (https://elv.sh/ref/builtin.html#src)